x/cork/types/msgs.go

```diff
@@ -62,3 +64,53 @@ func (m *MsgSubmitCorkRequest) MustGetSigner() sdk.AccAddress {
 	}
 	return addr
 }
+
+//////////////////////////
+// MsgScheduleCorkRequest //
+//////////////////////////
+
+// NewMsgScheduleCorkRequest return a new MsgScheduleCorkRequest
+func NewMsgScheduleCorkRequest(body []byte, address common.Address, blockHeight uint64, signer sdk.AccAddress) (*MsgScheduleCorkRequest, error) {
+	return &MsgScheduleCorkRequest{
+		Cork: &Cork{
+			EncodedContractCall:   body,
+			TargetContractAddress: address.String(),
+		},
+		BlockHeight: blockHeight,
+		Signer:      signer.String(),
+	}, nil
+}
+
+// Route implements sdk.Msg
+func (m *MsgScheduleCorkRequest) Route() string { return ModuleName }
+
+// Type implements sdk.Msg
+func (m *MsgScheduleCorkRequest) Type() string { return TypeMsgScheduleCorkRequest }
+
+// ValidateBasic implements sdk.Msg
+func (m *MsgScheduleCorkRequest) ValidateBasic() error {
```

Comment:
We may also want to call `ValidateBasic` on the embedded cork object.

Reply:
I don't believe that there is a ValidateBasic for a cork. What would it check?